admin_manual/configuration/server/occ_command.rst

```diff
@@ -1886,17 +1889,64 @@ You may also use ``password-from-env`` to reset passwords:
    Successfully reset password for layla
    
 Deleting A User
-^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~
 
 To delete a user, you use the ``user:delete`` command, as in the example below:
 
 ::
 
  sudo -u www-data php occ user:delete fred
-   
-   
+
+Expiring a User's Password
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note:: This command is only available when `the Password Policy app`_ is installed.
+
+::
+
+    user:expire-password <uid> [<expiredate>]
+
+To expire a user's password at a specific date and time, use the ``user:expire-password`` command.
+The command accepts two arguments, the user's uid and an expiry date.
+The expiry date can be provided using any of `PHP's supported date and time formats`_.
+It defaults to the previous day if it is not supplied.
+
+After the command completes, console output, similar to that below, confirms when the user's password is set to expire.
+
+::
+
+  The password for frank is set to expire on 2018-07-12 13:15:28 UTC.
+
+Command Examples
+^^^^^^^^^^^^^^^^
+
+::
+
+  # Expire the user "frank"'s password 24 hours ago.
```

Comment:
password 24 hours ago: this sounds so strange...
Wouldn´t it be better to say immediately ?

Reply:
I'm finding that bit of text hard to get just right. If you don't supply an expiry date, then the password will be set as being expired 24 hours prior to the time that the command was run.

Reply:
Then I would write that: ``the password will be set as being expired 24 hours prior to the time that the command set the date originally``. (or something like that)

For admins, one command example would be very benificial: expire now - how to do that...